boards/common/iotlab/include/board_common.h

```diff
@@ -49,46 +49,46 @@ extern "C" {
  * Tell the xtimer that we use a 16-bit peripheral timer
  * @{
  */
-#define XTIMER_WIDTH        (16U)
-#define XTIMER_OVERHEAD     (6U)
-#define XTIMER_SHOOT_EARLY  (3U)
+#define XTIMER_WIDTH              (16U)
```

Comment:
this is inconsistent with other changes here, and also with existing configs in RIOT  which are defined without `U`, hence signed.

Reply:
below, too

Reply:
Fixed